src/components/timecard/edit-shift-hours/EditShiftHours.jsx

```diff
@@ -140,16 +156,54 @@ const EditShiftHours = ({
           setShowEditShiftHours(false);
         }
       },
+      true,
       handleServerValidationErrors
     );
   };
 
+  const getCombinedErrors = () => {
+    const existingErrors =
+      Object.keys(errors).length > 0 ? errors : summaryErrors;
+
+    const combinedErrors = { ...existingErrors };
+
+    var timeInputErrors = (
+      <TimeInputErrors
+        clashingProperty={clashingProperty}
+        clashes={clashingTimes}
+      />
+    );
+
+    if (clashingProperty === 'startAndEndTime') {
+      combinedErrors['shift-start-time'] = {
+        message: timeInputErrors,
+      };
+      combinedErrors['shift-finish-time'] = {
```

Comment:
Per other constants comment, consider putting these control names in a constants file. They are now getting used in many places
`shift-start-time`, `shift-finish-time`

Reply:
👍 done